scripts/tests/homepage-release-contract.test.mjs

```
@@ -24,3 +24,31 @@ test('homepage release evidence is portable and records the verified ship state'
   assert.equal(evidence.decision, 'ship')
   assert.ok(evidence.artifacts.every((artifact) => !artifact.path_or_url.startsWith('file:')))
 })
+
+test('portrait proof overlay is bounded by the card at narrow widths', async () => {
+  const homepage = await readRepoFile('components/home/FrankXProductionHome.tsx')
+  const overlayMarkup = homepage.match(
+    /<div\b(?=[^>]*\bdata-home-proof-overlay\b)[^>]*>[\s\S]*?<\/div>/,
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)